tests/test_eload_submission.py

```diff
@@ -33,6 +33,14 @@ def tearDown(self):
             if os.path.exists(file_path):
                 os.remove(file_path)
 
+    def test_create_log_file(self):
+        logfile_name = os.path.join(self.eload.eload_dir,"submission.log")
+        assert os.path.exists(logfile_name)
+        self.eload.info("Testing the creation of logging file")
+        with open("logfile_name", "r") as test_logfile:
+            assert test_logfile.readline().startswith("Testing the creation of logging file")
+
+
```

Comment:
To be complete we should add another test where you create a second ELOAD object with the same eload number and show that the logging message are only written once.

Reply:
I have done the changes...but have to see if it works!